photon-api/src/main/scala/com/linkedin/photon/ml/supervised/classification/LogisticRegressionModel.scala

```diff
@@ -75,7 +132,7 @@ class LogisticRegressionModel(override val coefficients: Coefficients)
 object LogisticRegressionModel {
 
   /**
-   * Create a new logistic regression model with the provided coefficients (means) and variances
+   * Create a new logistic regression model with the provided coefficients (means) and variances.
    *
    * @param coefficients The feature coefficient means and variances for the model
```

Comment:
Isn't the usual name just "coefficient"? I don't know if "feature coefficient" is found in the literature.